### PR TITLE
Package nuget

### PR DESCRIPTION
**Pull request by mehagar**

### CODE REVIEW

**Comment by mehagar** [comment]
I confirmed that the LVjs passes with the published nuget.

File: vireoSDK.props [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <vireo_InstallDir>$(MSBuildThisFileDirectory)..\nipkgcore\</vireo_InstallDir>
```
Comment (rajsite) [inline vireoSDK.props]: This looks sketchy to me. We are making a nuget package for Vireo. It should be generic and just include the vireo.exe as content files. This is assuming it is being used in the build system and assuming some filesystem layout.

File: VireoSDK.nuspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/ni/VireoSDK/blob/incoming/LICENSE.txt</licenseUrl>
+    <projectUrl>https://github.com/ni/VireoSDK/</projectUrl>
+    <description>VireoSDK provides a subset of LabVIEW runtime functionality for smaller targets.</description>
+  </metadata>
+  <files>
+    <file src="dist/Debug/esh.exe" target="content/TestFiles" />
+    <file src="vireoSDK.props" target="build" />
+  </files>
+</package>
```
Comment (cglzaguilar) [inline VireoSDK.nuspec]: Add a newline to the end of the file to remove the red artifact that appears in online version of the diff at the end of the file.

File: vireoSDK.props [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <vireo_InstallDir>$(MSBuildThisFileDirectory)..\content\</vireo_InstallDir>
+    </PropertyGroup>
+</Project>
```
Comment (cglzaguilar) [inline vireoSDK.props]: Add a newline to the end of the file to remove the red artifact that appears in online version of the diff at the end of the file.

File: VireoSDK.nuspec (hunk 2 above)
Comment (mehagar) [reply VireoSDK.nuspec]: Done.

File: vireoSDK.props (hunk 3 above)
Comment (mehagar) [reply vireoSDK.props]: Done.

**Comment by rajsite** [comment]
@mehagar please address the build failure to continue

**Comment by rajsite** [comment]
@mehagar in the future please include a pull request description

**Pull request merged**